### DISCUSSION TITLE
Good Grief options for the website

### DISCUSSION THREAD

**Discussion by Arcadius2006** [LBPUnion/ProjectLighthouse#621] [Ideas]
Allow people to report users, levels, screenshots, playlists, reviews or comments via the website, this could be done with a button with a flag icon on it next to the website. After clicking, it would take you to a page where you can describe what the issue is with text and give it a tag

**Comment by Arcadius2006** [comment]
WHY IS THERE AN UPVOTE

**Reply by indo2042** [reply]
that was me🧌
